ui/src/app/utils/resourceActions.ts

```diff
@@ -0,0 +1,44 @@
+  export const NotebookActionList = [
```

Comment:
nit: would lowercase these since they aren't classes anymore. We do this elsewhere for exported constants